.github/workflows/tests.yml

```diff
@@ -49,18 +51,18 @@ jobs:
             runs-on: ubuntu-22.04
             fuse_package: libfuse3-dev
           # fuse-t
-          - cc: gcc-13
-            cxx: g++-13
+          - cc: gcc
+            cxx: g++
```

Comment:
@mpartel I re-checked that (both macos-13 and) macos-15 images have `gcc-13` installed and available (https://github.com/actions/runner-images/blob/main/images/macos/macos-15-Readme.md) and so I would like to vote for revert because it makes things clear.